boot/boot.js

```diff
@@ -85,6 +85,15 @@ $tw.utils.each = function(object,callback) {
 };
 
 /*
+Pushes a value to an array only when not yet contained.
+*/
+$tw.utils.pushOnce = function(array,value) {
+	if(0 > array.indexOf(value)){
```

Comment:
This should be changed to `array.indexOf(value) == -1` for consistency with existing code.